packages/suite-desktop/src-electron/app.ts

```diff
@@ -8,7 +8,7 @@ import { ipcMain } from './typed-electron';
 // eslint-disable-next-line @typescript-eslint/prefer-ts-expect-error
 // @ts-ignore
 import { SENTRY_CONFIG } from '@suite-config';
-import { isDev } from '@suite-utils/build';
+import { isDev } from '@suite-common/suite-utils';
```

Comment:
nit: I would probably prefer `isDevEnv` because it's more explicit name, but definitely not a big a deal

Reply:
Renamed, squashed, rebased to current develop